migrations/sql/V2019.03.12_1523__InitialSchema.sql

```diff
@@ -0,0 +1,46 @@
+CREATE TABLE "orderlyweb_user" (
```

Comment:
I think I've missed out on a key conversation about this since we last spoke - would you mind making a small markdown file explaining how these tables fit together?

Reply:
I'll add a bit to the readme file if that's ok